sdp/processors/datasets/lhotse.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+# To convert mp3 files to wav using sox, you must have installed sox with mp3 support
+# For example sudo apt-get install libsox-fmt-mp3
+from sdp.processors.base_processor import BaseProcessor
+
+
+class LhotseImport(BaseProcessor):
+    """Processor to create an initial manifest import from a Lhotse CutSet.
+
+    Lhotse is a library for speech data processing and loading; see:
+    - https://github.com/lhotse-speech/lhotse
+    - https://lhotse.readthedocs.io
+
+    TODO: list limitations
+
+    Returns:
+        This processor generates an initial manifest file with the following fields::
+
+            {
+                "audio_filepath": <path to the audio file>,
+                "duration": <duration of the audio in seconds>,
+                "text": <transcription (with capitalization and punctuation)>,
+            }
+    """
+
+    def process(self):
+        from lhotse import CutSet
```

Comment:
Is **lhotse** the only dependency that needs to be installed? Maybe provide a string on how to install **lhotse** if it is not?

Reply:
Yes, it only requires `pip install lhotse`. I'll update the doc accordingly.